src/fragments/start/getting-started/next/api.mdx

```diff
@@ -89,16 +95,18 @@ amplify push
 
 
 API key configuration
-✔ Enter a description for the API key: ·
-✔ After how many days from now the API key should expire (1-365): · 7
-GraphQL schema compiled successfully.
-
-? Do you want to generate code for your newly created GraphQL API Yes
-? Choose the code generation language target javascript
-? Enter the file name pattern of graphql queries, mutations and subscriptions src/graphql/**/*.js
-? Do you want to generate/update all possible GraphQL operations - queries, mutations and subscriptions Yes
-? Enter maximum statement depth [increase from default if your schema is deeply nested] 2
+✔ Enter a description for the API key: · (Just hit enter)
```

Comment:
for all of these `(Just hit enter)` would it not be better to do `(default)` or does it not suggest a default here?

Reply:
It doesn't necessarily spell out a default, and the docs just have that dot/apostrophe that may be hard to see.  

But (default) should work just fine and I'll make that change.